CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Mosaic JS Change Log
+
+## Version 0.10.0 ⚓️ Anchor
+
+<!-- [**Release 0.10.0, (<date-here>)**](https://github.com/OpenSTFoundation/mosaic.js/releases/tag/0.10.0) -->
+
+You can use mosaic.js to interact with contracts on mosaic chains.
+
+It requires [mosaic-contracts](https://github.com/OpenSTFoundation/mosaic-contracts) for ABIs and BINs of contracts.
+
+### Notable Changes
+
+* Library is provided via npm package [@openstfoundation/mosaic.js](https://www.npmjs.com/package/@openstfoundation/mosaic.js)
+* Now provides a `Mosaic` "super" provider to access both chains and their contract addresses ([#44](https://github.com/OpenSTFoundation/mosaic.js/pull/44)).
+* Now provides contract classes to interact with the contracts:
+  * Facilitator ([#42](https://github.com/OpenSTFoundation/mosaic.js/pull/42), [#51](https://github.com/OpenSTFoundation/mosaic.js/pull/51), [#56](https://github.com/OpenSTFoundation/mosaic.js/pull/56)).
```

Comment:
The facilitator is not exactly a contract interact.
I mean it interacts with the contract but it has additional logic to facilitate stake&mint and redeem&unstake process.
Maybe we can mention it as a sperate bullet point.

Reply:
Right. Will update.